Server/server.h

```diff
@@ -8,7 +8,28 @@
 #include "../Common/common.h"
 
 void send_nok();
-void ask_areas();
-void del_area();
+
+/**
+ * @fn void send_ok(pid_t sender, int num)
+ * @brief Send a OK message to the sender
+ * @param sender: pid of the sender
+ * @param num: id of the message queue
```

Comment:
It should definitely be named a different way, unless you have a good argument on why `num` is a good name